modules/phase_field/examples/ebsd_reconstruction/IN100-111grn.i

```diff
@@ -123,17 +135,11 @@
   [../]
   [./grain_tracker]
     type = GrainTracker
-    threshold = 0.1
-    convex_hull_buffer = 0.0
-    use_single_map = false
-    enable_var_coloring = true
-    condense_map_info = true
-    connecting_threshold = 0.05
-    execute_on = 'initial timestep_end'
-    flood_entity_type = ELEMENTAL
-    halo_level = 2
-    bubble_volume_file = IN100-grn-vols.txt
+    threshold = 0.2
+    connecting_threshold = 0.2
+    execute_on = 'initial timestep_begin'
```

Comment:
Sorry for the late reply.  That dataset has several split grains in it.  I
believe it needed unique values for both parameters to ensure that the
split regions of the same grain were allocated to the same order parameter
and grain number.

On Wed, Jul 13, 2016 at 12:32 PM, Michael Tonks <notifications@github.com>
wrote:

> In modules/phase_field/examples/ebsd_reconstruction/IN100-111grn.i
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_idaholab_moose_pull_7320-23discussion-5Fr70682597&d=CwMCaQ&c=54IZrppPQZKX9mLzcGdPfFD1hxrcB__aEkJFOKJFd00&r=cfRlC-bkBYcDcmBlxJT86aA-h_dr2ZJ37fRGraHZxBw&m=FrSpNjsmTHtkd7kCxaLzqp4K9U70GtK_pejYO0PTvlk&s=g7jXnU4taUuemHg8_0IRbPD-Cl9RT6nCKTeh509hHwQ&e=>
> :
>
> > @@ -123,17 +135,11 @@
> >    [../]
> >    [./grain_tracker]
> >      type = GrainTracker
> > -    threshold = 0.1
> > -    convex_hull_buffer = 0.0
> > -    use_single_map = false
> > -    enable_var_coloring = true
> > -    condense_map_info = true
> > -    connecting_threshold = 0.05
> > -    execute_on = 'initial timestep_end'
> > -    flood_entity_type = ELEMENTAL
> > -    halo_level = 2
> > -    bubble_volume_file = IN100-grn-vols.txt
> > +    threshold = 0.2
> > +    connecting_threshold = 0.2
> > +    execute_on = 'initial timestep_begin'
>
> For some reason, on this file only, I have to have both threshold values
> for it to run without getting an error. The error typically comes after the
> third time step.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_idaholab_moose_pull_7320_files_89dca26545e43aa9f6cab5da34cdec18c2e7fab9-23r70682597&d=CwMCaQ&c=54IZrppPQZKX9mLzcGdPfFD1hxrcB__aEkJFOKJFd00&r=cfRlC-bkBYcDcmBlxJT86aA-h_dr2ZJ37fRGraHZxBw&m=FrSpNjsmTHtkd7kCxaLzqp4K9U70GtK_pejYO0PTvlk&s=e7EuZCnHlUbzFxe9n4aM_5bMZGxSVF6ED1hsS0dwxzs&e=>,
> or mute the thread
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe_AHyuDuc4ewBEOFTa3UX4DB3JNxLkKZppks5qVS9UgaJpZM4JK3zV&d=CwMCaQ&c=54IZrppPQZKX9mLzcGdPfFD1hxrcB__aEkJFOKJFd00&r=cfRlC-bkBYcDcmBlxJT86aA-h_dr2ZJ37fRGraHZxBw&m=FrSpNjsmTHtkd7kCxaLzqp4K9U70GtK_pejYO0PTvlk&s=I_34cmDMUMBflfzLJOjOSNdmynDcwfCC9I-gOwtapt4&e=>
> .
>